rama-net/src/client/conn.rs

```diff
@@ -81,3 +83,113 @@ where
         self.serve(ctx, req)
     }
 }
+
+/// Mock connector can be used in tests to simulate connectors so we can test client and servers
+/// without opening actuall connections
+pub struct MockConnectorService<S> {
```

Comment:
Can you please move this to some kind of module which name's makes it clear that this is a test utility.